### ISSUE TITLE
When Toggle Continuous View is OFF

### ISSUE THREAD

**Issue by GaramGor** [sumatrapdfreader/sumatrapdf#2675]
When Toggle Continuous View is OFF the program ALWAYS freezes and breaks when navigating through a table of contents or a footnote.
I did check for version 3.4.4 and 3.4.5
3.4.3 work fine.

**Comment by kjk** [comment]
Can you provide a test document that shows the problem?

Does it happen for all documents or just one?

**Comment by GaramGor** [comment]
I tested on EPUB and FB2 files.
[test books.zip](https://github.com/sumatrapdfreader/sumatrapdf/files/8842511/test.books.zip)

**Comment by kjk** [comment]
I can't reproduce the issue.

I tried both files in 3.4.5.

I opened them, navigated using table of contest, changed continuous view with 'c', navigated again.

I didn't see a freeze.

**Comment by GaramGor** [comment]
Occurs only if Toggle Continuous View is OFF
If change to ON (checked) -all good

**Comment by GaramGor** [comment]
clarification
error occurs if set: zoom - fit content
PS: I can record video, if you want.

**Comment by kjk** [comment]
Yes, please post step-by-step instructions on how to reproduce and a video.

**Comment by kjk** [comment]
Nevermind, I can reproduce this (fit content AND non-continuous).

It's a deadlock I introduce to fix a crash.
